src/runtime/scss/tabs.scss

```diff
@@ -1,7 +1,7 @@
 .yfm-tabs {
     margin-bottom: 15px;
 
-    ol {
+    .yfm:not(.yfm_no-list-reset) & ol {
```

Comment:
Is it a unique and well-known CSS class? Should we add docs about it?

Reply:
This CSS class is used in [yfm-transform styles](https://github.com/yandex-cloud/yfm-transform/blob/3cd3ab0663d2bf640731c1fdd4d47c8c5382a05e/src/scss/_common.scss#L285)

I don't know if we should add docs about this class. But if it is necessary, then it should probably be added to yfm-transform docs

Reply:
Okay. Got it. I think we should not add it docs.